### PR TITLE
update controller: add owns

### PR DESCRIPTION
**Pull request by wanglei4687**
Signed-off-by: wanglei4687 <wanglei4687@gmail.com>

**What type of PR is this?**

- [ ] API-change
- [ ] BUG
- [x] Improvement
- [ ] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

**Which issue(s) this PR fixes:**

Fixes #

**What this PR does / why we need it:**

default description for owns
```
Owns defines types of Objects being *generated* by the ControllerManagedBy, and configures the ControllerManagedBy to respond to
create / delete / update events by *reconciling the owner object*.  This is the equivalent of calling
Watches(&source.Kind{Type: <ForType-forInput>}, &handler.EnqueueRequestForOwner{OwnerType: apiType, IsController: true})
```
**Special notes for your reviewer:**

Not Available

**Additional documentation (e.g. design docs, usage docs, etc.):**

Not Available

### CODE REVIEW

**Comment by wanglei4687** [comment]
@mergify update

File: pkg/controllers/matrixone/matrixonecluster_controller.go [hunk 1]
```diff
@@ -75,6 +78,9 @@ func (r *ClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 func (r *ClusterReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&v1alpha1.MatrixoneCluster{}).
+		Owns(&corev1.Service{}).
+		Owns(&appsv1.StatefulSet{}).
+		Owns(&policy.PodDisruptionBudget{}).
```
Comment (aylei) [inline pkg/controllers/matrixone/matrixonecluster_controller.go:83]: Should we watch PDB now? Looks like cluster controller does not create any

**Review by aylei — COMMENTED** [review]
The rest LGTM

File: pkg/controllers/matrixone/matrixonecluster_controller.go (hunk 1 above)
Comment (wanglei4687) [reply pkg/controllers/matrixone/matrixonecluster_controller.go]: yeah, I removed it. adress in https://github.com/matrixorigin/matrixone-operator/pull/116/commits/5f18ab4a2d05dc764c4a3b60762da33d3b584128

**Review by aylei — APPROVED** [review]
lgtm

**Pull request merged**